### PR TITLE
debian-control: python-imaging dependency now called python-pil

### PR DESCRIPTION
**Pull request by philipwinklerdelmic**

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -26,7 +26,7 @@ Depends: python (>= 2.7.1),
          python-serial,
          python-netifaces,
 # PIL is only needed in test cases, and indirectly by scipy for some image processing
-         python-imaging,
```
Comment (philipwinklerdelmic) [inline debian/control]: It was called python-pil in 16.04 already, right? Why isn't python-pil in the list?

**Comment by philipwinklerdelmic** [comment]
Could you also add a release file to the normal repository (not odemis-proposed)? Otherwise, it's really hard for me to test things.

File: debian/control (hunk 1 above)
Comment (pieleric) [reply debian/control]: This was to support 12.04, which we have supported until Odemis 2.9 (ie, the latest stable version) :-) 
Now that the next release will not need to support 12.04, we can finally change it. We could have different control files for each Ubuntu release that we support, but unfortunately with the standard Ubuntu (and Debian) build system, it's a pain.

**Comment by pieleric** [comment]
The normal repo has a Bionic branch since a couple of weeks already. Or do you mean something else?

**Pull request merged**